src/app/forgot-password/page.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Link from "next/link";
+
+import FormComponent from "./FormComponent";
+
+const UserRegisterPage = () => {
+
+  return (
+    <div className={`nc-PageSignUp `} data-nc-id="PageSignUp">
+      <div className="container mb-8 lg:mb-32">
+      <div className='mt-8 mb-16 flex items-center justify-between'>
+        <h2 className=" pl-6 flex items-center text-2xl leading-[115%] md:text-5xl md:leading-[115%] font-bold text-black dark:text-neutral-100 justify-center">
+          Recuperar <br/>
```

Comment:
What is broke line here?
```suggestion
          Recuperar
```

Reply:
its only for design